stl/inc/format

```diff
@@ -1410,13 +1410,17 @@ _NODISCARD constexpr basic_format_arg<_Context> _Get_arg(const _Context& _Ctx, c
     return _Arg;
 }
 
+template <class _Ty>
+inline constexpr bool _Is_standard_signed_or_unsigned_integer =

```

Comment:
This is missing four standard signed or unsigned integer types. I understand why, but I think we need a different name to avoid inadvertent reuse.